<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisDialog</class>
 <widget class="QDialog" name="AnalysisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализ по тегам</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxDateSelection">
     <property name="title">
      <string>Период анализа</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutDates">
      <item>
       <layout class="QHBoxLayout" name="layoutSingleDate">
        <item>
         <widget class="QRadioButton" name="radioButtonSingleDate">
          <property name="text">
           <string>Одна дата</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEditSingle">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutRange">
        <item>
         <widget class="QRadioButton" name="radioButtonDateRange">
          <property name="text">
           <string>Диапазон дат</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEditFrom">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTo">
          <property name="text">
           <string>до</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEditTo">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonAnalyze">
     <property name="text">
      <string>Анализ</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidgetSummary">
       <property name="columnCount">
        <number>2</number>
       </property>
       <column>
        <property name="text">
         <string>Тег</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Доля времени (%)</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
